packages/modal/package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "pcln-modal",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
We probably don't need to upgrade the version number since lerna should do that automatically.  Otherwise this PR LGTM.

Reply:
```suggestion
    "version": "2.0.0",
```